webpack/webpack.config.common.js

```diff
@@ -0,0 +1,31 @@
+const path = require("path");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+
+module.exports = {
+  entry: "./src/index.jsx",
+  module: {
+    rules: [
+      {
+        test: /\.(js|jsx)$/,
+        exclude: /(node_modules|bower_components)/,
+        loader: "babel-loader",
+        options: {
+          presets: ["@babel/env"]
+        },
+      },
+      {
+        test: /\.css$/,
+        use: ["style-loader", MiniCssExtractPlugin.loader, "css-loader"]
+      },
+    ],
+  },
+  resolve: { extensions: ["*", ".js", ".jsx"] },
+  output: {
+    path: path.resolve(__dirname, "dist/"),
+    publicPath: "/dist/",
+    filename: "[name].bundle.js",
+  },
+  plugins: [
+    new MiniCssExtractPlugin()
```

Comment:
Please take a look at this plugin https://www.npmjs.com/package/clean-webpack-plugin
It is used to clean/remove dist folder before webpack will build your code

Reply:
Thank, I will.